spec/rails_helper.rb

```diff
@@ -16,6 +16,10 @@
 
 require 'rspec/rails'
 
+SimpleCov.start do
+  add_filter 'app/uploaders/application_uploader'
```

Comment:
Why do we need it?

Reply:
I think that we can easily test our uploader 
https://shrinerb.com/docs/testing

Reply:
Probably, we cann`t test `application_uploader` becouse it very common uploader and we have no one entity with a field `image`

Reply:
I suppose that such kind of tests should be ok for us 
https://shrinerb.com/docs/testing#unit-tests

Reply:
Sorry, looks like I missed something, everything is okay 👌